### ISSUE TITLE
Http-Request with Type HEAD runs into Timeout

### ISSUE THREAD

**Issue by thomasblass** [neos/flow-development-collection#992]
### Description

When executing a HTTP-Request with type HEAD to an elasticsearch instance (Version 5.4+) curl waits for the content and runs into the timeout, even if the request ist completed.

We are using Flowpack/Elasticsearch for the connection. This package worked well until recently, when it was no more possible to create or delete an index. 

After debugging the issue, I came to the conclusion, that the problem lies in the usage of curl and HEAD request type. This problem was not visible since the last version of elasticsearch, where a change to the rest-api to fix the formaly malformed behavior occured. 
Here is the Issue describing the changes:
https://github.com/elastic/elasticsearch/pull/23112

In short, elasticsearch returns a content-length greater than 0, which I think causes curl to wait for the content which never arrives.

I will issue a Pull-Request to fix this.

#### Expected behavior
HTTP-Request with type HEAD executes and does not run into a timeout.

#### Actual behavior
The request runs into a timeout.

### Affected Versions
Flow: 4.0.5
